tests/dragonfly/replication_test.py

```diff
@@ -3190,18 +3194,26 @@ async def test_partial_replication_on_same_source_master_with_replica_lsn_inc(df
     await c_s2.execute_command(f"REPLICAOF NO ONE")
     # Make server 4 replica of server 2
     await c_s4.execute_command(f"REPLICAOF localhost {server2.port}")
+    await check_all_replicas_finished([c_s4], c_s2)
```

Comment:
TODO for #5580 

Remove this test all together. It was introduced as part of https://github.com/dragonflydb/dragonfly/pull/5189 but it is no longer needed as we don't do partial sync anymore via the `incremental snapshot`

Reply:
nvm I rewrote it with a takeover instead, and kept the test to check errornous flow regardless